### ISSUE TITLE
Unable to use third party libraries in latest version of msvc

### ISSUE THREAD

**Issue by Moon-ChangHyun** [cnjinhao/nana#678]
Because things provided as external libraries, such as png, jpg, etc., were compiled with an older version (~msvc2017) of the compiler, link error c1047 occurs when building the application.

Since most GUI tasks require features such as jpg and png, the usability of the nana library has decreased significantly since support for msvc2017 was discontinued.

These third-party libraries also need to provide source code so that users can build them themselves, or static library files built in the latest version of msvc.

**Comment by cnjinhao** [comment]
Agreed, it will contain source code of third-party libraries in future release.
